src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -183,6 +183,10 @@ const useCloudEnvironmentPolling = googleProject => {
         Ajax(signal).Disks.list({ googleProject, creator: getUser().email, saturnApplication: tools.galaxy.appType }),
         Ajax(signal).Disks.list({ googleProject, creator: getUser().email, saturnApplication: tools.cromwell.appType })
       ]) : [[], [], [], []]
+      _.forEach(disk => disk.appType = tools.galaxy.appType)(galaxyDisks)
```

Comment:
@salonishah11 if you were to add "formattedBy" to the information returned by the Disks list method, then I could key off of that value instead of injecting appType here.

Reply:
If it is non-trivial, I will incorporate it in [my current PR](https://github.com/DataBiosphere/leonardo/pull/2401) 👍

Reply:
see above for another way to do this